src/main/java/blusunrize/immersiveengineering/common/IEContent.java

```diff
@@ -790,6 +791,8 @@ public static void init()
 		ThermoelectricHandler.registerSourceInKelvin(Blocks.PACKED_ICE, 240);
 		ThermoelectricHandler.registerSourceInKelvin(Blocks.BLUE_ICE, 200);
 		ThermoelectricHandler.registerSourceInKelvin(IETags.getTagsFor(EnumMetals.URANIUM).storage, 2000);
+		ThermoelectricHandler.registerSourceInKelvin(BlockTags.getCollection().getTagByID(new ResourceLocation("forge:ices/dry")), 150);
+		ThermoelectricHandler.registerSourceInKelvin(BlockTags.getCollection().getTagByID(new ResourceLocation("forge:storage_blocks/uraninite")), 2000);
```

Comment:
We have a utility function for this, `TagUtils.createBlockWrapper`

Reply:
Done